<launch>
  <param name="my_name" value="robot1" />
  <rosparam param="other_robots">['robot2']</rosparam>
  <rosparam param="caution_edges">['WayPoint5_WayPoint6_WayPoint7_WayPoint8']</rosparam>
  <node name="path_planner" pkg="dijkstra" type="path_planner.py" output="screen" />
</launch>
